src/main/kotlin/no/nav/syfo/domain/Vurdering.kt

```diff
@@ -14,10 +16,33 @@ data class Vurdering private constructor(
     val varsel: Varsel?,
     val document: List<DocumentComponent>,
     val journalpostId: String?,
-
 ) {
+
+    constructor(
+        personident: PersonIdent,
+        veilederident: String,
+        begrunnelse: String,
+        document: List<DocumentComponent>,
+        type: VurderingType,
+    ) : this(
+        uuid = UUID.randomUUID(),
+        personident = personident,
+        createdAt = nowUTC(),
+        veilederident = veilederident,
+        type = type,
+        begrunnelse = begrunnelse,
+        document = document,
+        journalpostId = null,
+        varsel = Varsel(),
```

Comment:
Skal vi lage `Varsel` for oppfylt og avslag?

Reply:
Good point, jeg endrer!

Reply:
Til `varsel: Varsel? = null,` nå